project-code/core/common/src/main/scala/play/core/server/servlet/Play2Server.scala

```diff
@@ -30,21 +32,21 @@ object Play2WarServer {
 
   var playServer: Option[Play2WarServer] = None
 
-  def apply(contextPath: Option[String] = None) = {
+  def apply(contextPath: Option[String] = None): Unit = {
```

Comment:
Your IDE may emit a warning about this without this change, but I would recommend not making sweeping changes like this across the code base or even just a single file when working on something that you want people to review, makes it very difficult for a reviewer to see what you've actually changed. If you want to add type annotations to every method, do it in a separate pull request.

Reply:
You are right, I started to do this in the first file and forgot to revert that. Will fix it.

Reply:
Just fixed it. :)